CHANGELOG.md

```diff
@@ -10,29 +10,24 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - An optional "stack identifier" that can be set during [initialization](README.md#initialization) so we can track libraries that use node-sparkpost via the User-Agent header by @ewandennis.
 
 ## [2.0.0] - 2016-11-04 - *breaking*
```

Comment:
it might be worth adding a human-readable paragraph here summarizing the changes and that it's a major version release, etc.